jpa/tck-dist/src/main/resources/eftl-1.1.adoc

```diff
@@ -0,0 +1,23 @@
+= Eclipse Foundation Technology Compatibility Kit License - v1.1
+
+The accompanying Technology Compatibility Kit is "Copyright (c) [$date-of-document], Eclipse Foundation AISBL and its licensors."
+
```

Comment:
According to [this document](https://www.eclipse.org/projects/handbook/#ip-copyright-headers),

> An Eclipse Foundation open source project is not a legal entity. Further, the Eclipse Foundation does not assume ownership of contributions (the Eclipse Foundation is only a copyright holder in those cases where an employee of the Eclipse Foundation has authored the contribution)

The copyright should not be held by the Eclipse Foundation (I do not know what AISBL is, though), but should be held by  Contributors to the Eclipse Foundation.

Reply:
> The copyright should not be held by the Eclipse Foundation (I do not know what AISBL is, though), but should be held by Contributors to the Eclipse Foundation.

Specification projects operations are diffrent from regular Eclipse open source projects.
Please see https://www.eclipse.org/projects/handbook/#specifications-efsl
```
The Eclipse Foundation takes copyright ownership of the specification document of every ratified Final Specification.
```

Reply:
> The copyright should not be held by the Eclipse Foundation (I do not know what AISBL is, though), but should be held by Contributors to the Eclipse Foundation.

https://www.eclipse.org/europe/aisbl-membership-faq/ mentions AISBL

https://logos-pa.com/insights/a-dive-into-the-world-of-associations states:
 
> Let’s start with the basics! The world of associations can be intimidating, especially if you are thinking of setting up a new one and you don’t know where to start. No need to panic, here are some essential tips.
ASBL or AISBL ?
>
> In Belgium, if you want to set up a non-profit association, you have the choice between ASBL and AISBL. The former is a group of natural or legal persons pursuing a non-profit purpose whereas the latter is a group of natural or legal persons pursuing a non-profit and international purpose. Before your head starts to ache, here are key considerations:

>    The ASBL is easier to set up and enables to save time and money. For example, there is no need for a Royal Decree. Nevertheless, the purpose of the association should stay within the remit of the Belgian territory… Besides, its governance structure is less flexible.
>
>    The AISBL appeals to founders with a far-reaching scope and a flexible structure. However, the process of setting it up is more burdensome since it requires a deed drawn up by a notary and recognised by a Royal Decree.